apps/web/src/components/Composer/Actions/Attachment.tsx

```diff
@@ -83,23 +81,13 @@ const Attachment: FC<Props> = ({ attachments, setAttachments }) => {
       const { files } = evt.target;
       // Count check
       if (files && (hasVideos(files) || (isImageType(files) && files.length + attachments.length > 4))) {
-        return toast.error('Please choose either 1 video or total 4 photos.');
+        return toast.error('Please choose either 1 video or up to 4 photos.');
```

Comment:
@bigint we allow either 1 video or 4 photos? or do we allow 3 pics too?

Reply:
right now 1 video or 4 photos+gif